src/PSDocs.Azure/docs/Azure.Template.Doc.ps1

```diff
@@ -20,15 +20,18 @@ function global:GetTemplateParameter {
                 Description = ''
                 DefaultValue = $Null
                 AllowedValues = $Null
+                Required = "Required"
```

Comment:
@vicperdana Potentially this can be localized as well since it will be part of the output.

Reply:
@BernieWhite just to clarify, would this be related to:
`$LocalizedData.Required` in line number 238.

Reply:
@vicperdana More suggesting that we `Required` now `Yes` / `No` can be localized.

So at some point, if someone wants to localize PSDocs.Azure then Yes/ No can be specified in another language.

I've added the suggestion as a commit.